src/components/modules/AuthHeader/AuthHeader.module.scss

```
@@ -0,0 +1,48 @@
+@import "@sass/_colors.scss";
+@import "@sass/_variables.scss";
+@import "@sass/_media.scss";
+
+.container {
+    display: flex;
+    align-items: center;
+    padding: 2rem 4rem;
+    color: $primary-dark;
+    background-color: $page-background-color;
+
+    .backBtnContainer {
+        position: absolute;
+        display: flex;
+        align-items: center;
+        gap: 0.2rem;
+        cursor: pointer;
+
+        .arrowLeftIcon path {
+            fill: $primary-dark;
+        }
+
+        .backBtnText {
+            color: $primary-dark;
+            border-bottom: 0.1rem $primary-dark solid;
```

for borders, box-shadows we're sticking to px
```suggestion
            border-bottom: 0.1px $primary-dark solid;
```

---

replaced